src/main/webapp/completedPath.html

```diff
@@ -21,11 +21,10 @@ <h2 class="step-outline">THE END?</h2>
 
   <div class="step-wrap, step-dialogue">
       <div>
-          <!--TODO:add graduation speech as dialogue-->
-          <p id="dialogue-container"></p>
+          <p id="dialogue-containerX"></p>
```

Comment:
[something]X is not really a descriptive/ meaningful naming scheme.

Reply:
+1